webapp/channels/src/components/channel_invite_modal/group_option/group_option.test.tsx

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import type {ComponentProps} from 'react';
+import React from 'react';
+import {IntlProvider} from 'react-intl';
+import {Provider} from 'react-redux';
+
+import type {Group} from '@mattermost/types/groups';
+
+import store from 'stores/redux_store';
+
+import type {Value} from 'components/multiselect/multiselect';
+
+import {renderWithContext, waitFor} from 'tests/react_testing_utils';
+
+import GroupOption from './group_option';
+
+const mockGroup = {
+    id: 'group-id',
+    display_name: 'Group Name',
+    name: 'groupname',
+    member_ids: ['user1', 'user2'],
+    member_count: 2,
+} as Group&Value;
+
+describe('GroupOption', () => {
+    let props: ComponentProps<typeof GroupOption>;
```

Comment:
```suggestion
    let props: Props;
```

you can import Props directly from `group_option`